cypress/page/menu-content.page.ts

```diff
@@ -3,7 +3,8 @@ class MenuContentPage {
     private menuContentPageURL: string
 
     constructor() {
-        this.tShirtMenu = "#block_top_menu > ul > li:nth-child(3) > a";
+    
+        this.tShirtMenu = "#block_top_menu > .sf-menu > li > a[title='T-shirts']";
```

Comment:
you can preserve the ul, since is more logic than the .sf-menu:
`#block_top_menu > ul > li > a[title='T-shirts']`